website/templates/website/error_pages/404.html

```diff
@@ -2,8 +2,17 @@
 
 {% block content_body %}
 
-<!-- Content -->
-<div class="content-card">
-  <div class="page-heading"><h2>404 Not Found</h2></div>
-</div>
+<head>
+    <link href="https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@600;900&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/4b9ba14b0f.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" type="text/css" href="{% static "css/styles404.css" %}>
```

Comment:
This part of the code should be in the `{% block extra_css %}{% endblock %}` on the top of the page